src/open_inwoner/scss/components/Plan/PlanFile.scss

```
@@ -1,9 +1,7 @@
-#document-create {
-  /// Make required asterisk visible for this component
-  .label__label--required {
-    color: var(--color-red-notification);
-    padding-left: var(--spacing-tiny);
-    //make asterisks invisible by default, only make them visible if component has the caption
-    display: inline;
+.plan {
+  .file-input-form {
+    .form__actions--fullwidth {
```

add these classes to one line, unnecessary nesting
```suggestion
    .plan .file-input-form .form__actions--fullwidth {
```